srcs/parse/map/parse_map.c

```
@@ -0,0 +1,417 @@
+	while (i < map_lines)
+	{
+		map_data->map[i] = calloc(max_len + 1, sizeof(char));
+		if (!map_data->map[i])
+		{
+			while (i > 0)
+				free(map_data->map[--i]);
+			free(map_data->map);
+			map_data->map = NULL;
+			return (-1);
+		}
+		while (j < max_len)
```

Cppcheck (2.18.0)

[error] Uninitialized variable

(legacyUninitvar)